identity-admin/webapp/pages/_app.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { UserProvider } from '@auth0/nextjs-auth0';
+import { AppProps } from 'next/app';
 
-// @todo.
-const App: React.FC<any> = ({ Component, pageProps }) => {
+function App({ Component, pageProps }: AppProps) {
```

Comment:
In VS Code I still get a ESLint/TypeScript error on this line:

```
Missing return type on function. 
eslint(@typescript-eslint/explicit-module-boundary-types)
```

```suggestion
function App({ Component, pageProps }: AppProps): JSX.Element {
```

Reply:
Ah, it's only appearing as a warning for me, but that's an easy enough change